packages/react/src/components/Campaign/Form.js

```diff
@@ -161,6 +166,8 @@ const Button = styled(_Button)`
   color: #fff;
   margin-top: 40px;
   width: auto;
+  text-transform: ${({ theme }) =>
+    theme.uppercaseTitles ? 'uppercase' : 'none'};
```

Comment:
I wouldn't use crane's theme in the react package. Let crane handle this internally, and react just consuming crane comps.

Reply:
We're already using the theme lots in that package. Maybe factoring it out is something for later.